contracts/libraries/Validator.sol

```diff
@@ -72,4 +31,97 @@ library Validator {
         }
         return computedHash == rootHash;
     }
+
+    // @param txHash      transaction hash
+    // @param rootHash    block header of the merkle tree
+    // @param input0      indicator for nonzero first input
+    // @param input1      indicator for nonzero second input
+    // @param sigs        transaction signatures
+    function checkSigs(bytes32 txHash, bytes32 confirmationHash, bool input0, bool input1, bytes sigs, bytes confirmSignatures)
+        internal
+        pure
+        returns (bool)
+    {
+        require(sigs.length % 65 == 0 && sigs.length == 130, "signatures must be 65 bytes in length");
+        require(confirmSignatures.length % 65 == 0, "signatures must be 65 bytes in length");
+        require(input0 || input1, "at least one of the inputs must be present");
+
+        
+        bytes memory sig0;
+        bytes memory sig1;
+        if (input1 && input0) {
+            require(sigs.length == 130 && confirmSignatures.length == 130,
```

Comment:
`sigs.length == 130` also redundant here.

Reply:
Changed in the refactor